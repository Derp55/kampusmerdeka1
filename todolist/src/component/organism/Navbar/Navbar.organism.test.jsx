import renderer from "react-test-renderer";
import Navbar from "./Navbar";

describe("Navbar", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<Navbar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
