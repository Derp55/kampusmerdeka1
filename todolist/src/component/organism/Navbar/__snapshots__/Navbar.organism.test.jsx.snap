// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Navbar > should render correctly 1`] = `
<div>
  Navbar
</div>
`;
